connectors/agentic-ai/element-templates/agenticai-aiagent-outbound-connector.json

```diff
@@ -185,11 +357,11 @@
     },
     "type" : "Dropdown",
     "choices" : [ {
-      "name" : "Default Credentials Chain (Hybrid/Self-Managed only)",
-      "value" : "defaultCredentialsChain"
-    }, {
       "name" : "Credentials",
       "value" : "credentials"
+    }, {
+      "name" : "Default Credentials Chain (Hybrid/Self-Managed only)",
+      "value" : "defaultCredentialsChain"
```

Comment:
💭 Interesting that the order changed here by removing the `permits`. No need for action, it's probably better this way. 🤔